individual/Core/TaxLogic/ca/TaxBrackets.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+class CA_TaxBrackets {
+  public static function getBrackets($year) {
+    $brackets = null;
+    if ($year === 2014 || $year === 2015)
+  	{
+  		$brackets = array(
+  			array('start' =>      0, 'stop' =>  44700, 'rate' => 0.15),
```

Comment:
There are *critical* errors in the code. All of your numbers are off by one, according to http://www.cra-arc.gc.ca/tx/ndvdls/fq/txrts-eng.html. For instance, the first bracket should be 0 to 44701, and the second should start at 44702. Please double check *all* numbers [like deductions and such]. The tax documentation is confusing, but, heck, "Blame Canada!" https://www.youtube.com/watch?v=bOR38552MJA